packages/components/navbar/src/Navbar.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Box,
+  type CommonProps,
+  type ExpandProps,
+  Flex,
+} from '@contentful/f36-core';
+import React from 'react';
+import { getNavbarStyles } from './Navbar.styles';
+
+type NavbarOwnProps = CommonProps & {
+  children?: React.ReactNode;
+  account?: React.ReactNode;
+  search?: React.ReactNode;
+  switcher?: React.ReactNode;
+  help?: React.ReactNode;
+  badge?: React.ReactNode;
+  /**
+   * Items that will be rendered on the bottom-right of the navbar.
+   * Useful for separating other navigation items from main ones, (e.g. separating "Settings" from all other navigation items).
+   */
+  bottomRightItems?: React.ReactNode;
```

Comment:
I'm wondering if we can come up with a semantic name here that gives greater understanding of what to put in there. `secondaryNavigationMenu` or something, maybe?

Reply:
I wouldn't use Menu in the name, since it can imply that it expects `Menu` or `MenuItems` that will be rendered in the `Menu`.

Maybe sth like:
- `bottomRightItems` -> `secondaryNavigationItems`/`secondaryNavItems`
-  `topRightItems` -> `contextualActions`

@fabe maybe you have some good suggestions?